session.go

```diff
@@ -296,6 +297,8 @@ func (session *Session) ReceiveWith(localClientID, remoteClientID string, buf []
 
 		if conn, ok := session.connections[connKey(localClientID, remoteClientID)]; ok {
 			conn.SendAck(packet.SequenceId)
+		} else {
```

Comment:
I would recommend this flow:
if !ok return err
return conn.SendAck
This way we also handle and return the sendack error

Reply:
Yes. I just checked, both heap.Push and conn.SendAck have no return value